Tests/SQLiteTests/SQLite3Tests.swift

```diff
@@ -63,6 +63,17 @@ class SQLite3Tests: XCTestCase {
             if let results = try database.execute("SELECT * FROM `foo`").first {
                 XCTAssertEqual(results.data["bar"], unicode)
             }
+            else {
+                XCTFail("The query should return a result.")
+            }
+            
+            if let results = try database.execute("SELECT * FROM `foo` WHERE bar = '\(unicode)'").first {
```

Comment:
Ok, so yes, I did that. However, shouldn't the unit tests be excluded from the coverage?